src/main/java/org/jfree/chart/date/SerialDate.java

```diff
@@ -394,10 +402,11 @@ public static String monthCodeToString(int month, boolean shortened) {
      * @return {@code -1} if the string is not parseable, the month of the
      *         year otherwise.
      */
-    public static int stringToMonthCode(String s) {
+    public static @IntRange(from = -1, to = 12) int stringToMonthCode(String s) {
 
         final String[] shortMonthNames = DATE_FORMAT_SYMBOLS.getShortMonths();
-        final String[] monthNames = DATE_FORMAT_SYMBOLS.getMonths();
+        @SuppressWarnings("index") // Need annotations on Date Format Symbols
```

Comment:
Is this necessary?  The calls to `getMonths` above didn't need `@SuppressWarnings`.